### PR TITLE
Grammar, styling, word wrap, contribs, sec. issues

### PR DESCRIPTION
**Pull request by dayton1987**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,40 +41,58 @@ settings] you may want:
...
+## Configuration
 
-There is only one - your public API key. You can get that by creating an
-account [Uploadcare]. You can use demo public key during dev stage, but note that
-demo account files are removed every few hours.
+### Required settings
 
+In fact, there's only one required setting — your public API key. In order to get the key
+you should register with Uploadcare and set up a project to store your files and settings.
+Check out our [knowledgebase][kb_intro] to get up an running with your Uploadcare account
```
Comment (dmitry-mukhin) [inline README.md]: knowledge base

File: README.md [hunk 2]
```diff
@@ -39,40 +41,58 @@ settings] you may want:
...
+in minutes. You can always use our demo public key during your dev stage. But keep in mind,
+files uploaded to the demo account are removed every few hours.
```
Comment (dmitry-mukhin) [inline README.md]: remove note on pub keys?

File: README.md [hunk 3]
```diff
@@ -174,9 +196,10 @@ A full list of tabs supported in the latest widget version
   </tr>
```
Comment (dmitry-mukhin) [inline README.md]: we've removed huddle

File: README.md (hunk 1 above)
Comment (dayton1987) [inline README.md]: Acknowledged

File: README.md (hunk 3 above)
Comment (dayton1987) [inline README.md]: For King and Country!

File: README.md (hunk 2 above)
Comment (dayton1987) [inline README.md]: Like, isn't it great to know you can always try something out without signing up? If they've got files of their own, it won't feel comfy to upload them to some public demo account anyway. U know wat I'm sayin?

**Comment by dmitry-mukhin** [comment]
@dayton1987 sup

**Pull request closed without merging**